### PR TITLE
Improving metric queries

### PR DESCRIPTION
**Pull request by aleksfront**
* Metric query improvements extracted from #1944
* Metrics added to namespaces and jobs
* Metric loaders are moved from stores to React components. The reason here because metrics doesn't really relate to stores. See this comment https://github.com/lensapp/lens/pull/1944#discussion_r556622245

node metrics

### CODE REVIEW

File: src/renderer/api/endpoints/metrics.api.ts
Comment (Nokel81) [inline src/renderer/api/endpoints/metrics.api.ts]: Shouldn't this really be `T extends IMetrics`?

**Comment by jakolehm** [comment]
Moving to 5.2.0 because it's an enhancement.

**Review by Nokel81 — APPROVED** [review]
Please do not merge, I'll merge this after the release of 5.1.3

**Pull request merged**
